### PR TITLE
Address some JSON-related ReSpec warnings

### PR DESCRIPTION
**Pull request by shs96c**
This change is [Reviewable](https://reviewable.io/reviews/w3c/webdriver/580)

### CODE REVIEW

File: webdriver-spec.html [hunk 1]
```diff
@@ -829,7 +829,7 @@
 
  <li><p>If <var>data</var> is not null, let <var>response</var>’s
   <a>body</a> be a JSON <a>Object</a> with a key <code>value</code>
-  set to the <a>JSON serialisation</a> of <var>data</var>.
+  set to the <a>JSON clone</a> of <var>data</var>.
```
Comment (andreastt) [inline webdriver-spec.html]: This should stay the same, I just need to get around to define _JSON serialisation_.

File: webdriver-spec.html [hunk 2]
```diff
@@ -3342,21 +3342,47 @@
...
  and <code>window.outerHeight</code> [[!DOM]] properties.
 
-<p>A <a>top-level browsing context</a>’s <dfn>window position</dfn>
- is defined as a dictionary of the <a>screenX</a> and <a>screenY</a> attributes
- on the <a><code>WindowProxy</code></a> it corresponds to.
- Its <a>JSON representation</a> is the following:
+<p>A <a>top-level browsing context</a>’s <dfn>window position</dfn> is
+ defined as a dictionary of the <a>screenX</a> and <a>screenY</a>
+ attributes on the <a><code>WindowProxy</code></a> it corresponds to.
+ The <dfn>JSON serialization of window position</dfn> is a
```
Comment (andreastt) [inline webdriver-spec.html]: We don’t _need_ a separate definition for each JSON serialisation if we think of it as a trait, but I guess this OK to make the link point to the right location.

File: webdriver-spec.html (hunk 1 above)
Comment (andreastt) [inline webdriver-spec.html]: What are we trying to do here? I think the data that comes out of each command is JSON conforming already?

**Comment by AutomatedTester** [comment]
I have created #585 which clears up some of the issues I saw in this PR. I did it earlier today but only getting chance to submit the PR now

File: webdriver-spec.html (hunk 1 above)
Comment (shs96c) [inline webdriver-spec.html]: Possibly, but the clone algorithm handles some edge cases we might miss, and ensures things like collections get handled nicely.

Comment (andreastt) [inline webdriver-spec.html]: Sure, but then we can avoid doing it for _Execute Script_ and _Execute Async Script_?

Comment (shs96c) [inline webdriver-spec.html]: This may not be entirely correct. Needs an edit

Comment (andreastt) [inline webdriver-spec.html]: I’m not super-fussed you need to fix that here. r+ on this from my end.

**Comment by shs96c** [comment]
A lot of these are covered by #585, or made irrelevant thanks to 17d09513c3ae399e3bed8cace24e3a61340bed54. Abandoning.

**Pull request closed without merging**